src/shipit_code_coverage/tests/test_github.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+
+import responses
+
+from shipit_code_coverage.github import GitHubUtils
+
+
+@responses.activate
+def test_get_commit(GITHUB_COMMIT):
+    gu = GitHubUtils('', '', '')
+    assert gu.get_commit(GITHUB_COMMIT) == GITHUB_COMMIT
```

Comment:
This should be `gu.get_commit(MERCURIAL_COMMIT) == GITHUB_COMMIT`. The get_commit function maps a Mercurial hash to a Git hash.
For `get_mercurial_commit` it's the other way around.

I know the tests work anyway, but like this they can be confusing to new contributors.

Reply:
Done. I had to put the calls to `responses.add` back into the `test_*` functions as `responses` has a weird behavior with multiple URLs (related to https://github.com/getsentry/responses/issues/170#issuecomment-350373745 I think). The tests would only pass if run at once.

Reply:
That comment shouldn't apply here, as the URLs are different in our case.
What was happening exactly with the responses.add calls in the fixtures?

Reply:
Here's the trace when running both tests: [trace.txt](https://github.com/mozilla-releng/services/files/1817072/trace.txt). Commenting out one of the tests would work fine.

Looks like at some point, responses forgets the response we've set and can't match it anymore.
I'm not sure how pytest fixtures work under the hood and how responses interfaces with them but [matches are only removed when there are multiple possible matches](https://github.com/getsentry/responses/blob/573e2be996c64c28ed67b303309d385579a090ae/responses.py#L513).

Reply:
Ah, maybe try not making the fixtures have `scope="session"`

Reply:
Argh, of course. Fixed.